components/landing/hero.tsx

```diff
@@ -1,3 +1,34 @@
+import { Button } from '../ui';
+import Link from 'next/link';
+
 export default function Hero() {
-  return <section>{/* put the title and subheading here */}</section>;
+  return (
+    <section>
+      {/* Hero Section Container */}
+      <div className="max-w-full h-[70vh] lg:min-h-screen flex flex-col justify-center items-center gap-6 md:gap-8">
+        {/* Primary Heading Container */}
+        <h1 className="flex flex-col justify-center items-center text-[40px] md:text-7xl font-medium leading-tight">
```

Comment:
- make header font thicker
```suggestion
        <h1 className="flex flex-col justify-center items-center text-[40px] md:text-7xl font-black leading-tight">
```